docs/singleKeyvault/README.md

```diff
@@ -0,0 +1,367 @@
+# Deploying Single Cluster with Keyvault 
+
+If you followed our first workload walkthrough you saw how Bedrock enables you to scaffold and generate Terraform deployment scripts. In the first workload walkthrough, to demonstrate simple GitOps workflow, we deployed a simple cluster using Azure Simple Terraform template. However in the real world, in order to have secure deployments, we need a way to store the essential secrets in Keyvault. In addition, we will also deploy a vnet to provide isolation to our cloud infrastructure. 
+
+In upcoming advanced challenges, we will be using the Bedrock automation to repeat the cluster creation process by scaffolding configurations to deploy multiple clusters. Since all these clusters use common resources like Keyvault, Storage Account and a Vnet, we will deploy these resources using azure-common-infra template. The environment provisioned using this template is a dependency for other environments (azure-single-keyvualt) we will be using in the subsequent walkthroughs.
+
+## Deplying the common infrastructure:
+
+Before attempting to deploy the infrastructure environments, you will also need to create an Azure Storage Account. You can do this in Azure Portal, or by using the Azure CLI:
+
+### Resource Group Requirement
+
+This environment requires a single resource group be created.  The requisite variable is `global_resource_group_name`.  To use the Azure CLI to create the resource group, see [here](https://github.com/microsoft/bedrock/blob/master/cluster/azure/README.md).
+
+To create a resource group, you can use the following command 
+
+```
+$ az group create -l westus2 -n my-global-rg
+```
+
+### Create Storage Account in Azure
+
+Before attempting to deploy the infrastructure environments, you will also need to create an Azure Storage Account. You can do this in Azure Portal, or by using the Azure CLI:
```

Comment:
nit: removed words
To deploy infrastructure environments, you will need to create an Azure Storage Account.